recipes/r-rsvg/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = '1.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rsvg
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: rsvg_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/rsvg_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/rsvg/rsvg_{{ version }}.tar.gz
+  sha256: 8ec457f1d52894619329650c58e2597b168e3312a1129d7863e23abad7d54e9e
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [osx]
```

Comment:
In the future, you can put this on one line with the selector `[win or osx]`.

Reply:
Thanks!